snakefiles/setup.smk

```diff
@@ -1,53 +0,0 @@
-# Setup pipeline constants by parsing filenames of raw read data
```

Comment:
File deleted: this dealt with raw reads, but similar logic now added to the main snakemake file to handle starting from existing alignments